crates/polars-core/src/chunked_array/comparison/mod.rs

```diff
@@ -759,6 +759,7 @@ fn struct_helper<F, R>(
     op: F,
     reduce: R,
     value: bool,
+    apply_null_validity: bool,
```

Comment:
nit: this is named `is_missing` in other helpers. Maybe that should be here as well. Not really a blocker though

Reply:
Sure, I will rename to `is_missing`